rivalcfg/devices/rival700.py

```diff
@@ -61,6 +68,28 @@
             "default": 1000,
         },
 
+        "logo_rgbgradient": {
+            "description": "Set the logo colors and effects",
+            "cli": ["-o", "--logo-colorshift"],
+            "command": [0x05, 0x00],
+            "report_type": usbhid.HID_REPORT_TYPE_FEATURE,
+            "value_type": "rgbgradientv2",
+            "rgbgradientv2_header": _RGBGRADIENTV2_HEADER,
+            "led_id": 0x0,
+            "default": _DEFAULT_RGBGRADIENTV2,
+        },
+
+        "wheel_rgbgradient": {
```

Comment:
```suggestion
        "wheel_color": {
```
This will replace the other method that supports only static colors, we do not need both :)